### PR TITLE
Added FF15IPQEXT

### PR DESCRIPTION
**Pull request by CarrieBrown**

### CODE REVIEW

File: projects/FF15IPQEXT.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Department: Chemistry
+Description: Parameterizing force fields for artificial amino acids through molecular dynamic simulations 
+FieldOfScience: Chemistry
+ID: '611'
+Name: FF15IPQEXT
+Organization: University of Pittsburg
```
Comment (brianhlin) [inline projects/FF15IPQEXT.yaml]:
```suggestion
Organization: University of Pittsburgh
```

File: projects/FF15IPQEXT.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Department: Chemistry
+Description: Parameterizing force fields for artificial amino acids through molecular dynamic simulations 
+FieldOfScience: Chemistry
+ID: '611'
+Name: FF15IPQEXT
```
Comment (brianhlin) [inline projects/FF15IPQEXT.yaml]: I thought we were using the `<Institution>_<PI Last name>` format?

File: projects/FF15IPQEXT.yaml (hunk 1 above)
Comment (CarrieBrown) [reply projects/FF15IPQEXT.yaml]: Good catch! I'll fix this.

File: projects/FF15IPQEXT.yaml (hunk 2 above)
Comment (CarrieBrown) [reply projects/FF15IPQEXT.yaml]: We are. This is a project that was created before this standardization was implemented but wasn't added to topology.

**Pull request merged**